python/langsmith/client.py

```diff
@@ -215,6 +216,24 @@ def _serialize_json(obj: Any, depth: int = 0) -> Any:
         return repr(obj)
 
 
+def _dumps_json(obj: Any) -> bytes:
+    """Serialize an object to a JSON formatted string.
+
+    Parameters
+    ----------
+    obj : Any
+        The object to serialize.
+    default : Callable[[Any], Any] or None, default=None
+        The default function to use for serialization.
+
+    Returns:
+    -------
+    str
+        The JSON formatted string.
+    """
+    return orjson.dumps(obj, default=_serialize_json)
```

Comment:
lets add some of the flags like serialize numpy etc etc

Reply:
is that applied before or after our default serialize json?

Reply:
before I think

Reply:
ah ya otherwise the passthrough options wouldn't make sense. thanks